src/code-examples/dialog/html.js

```diff
@@ -1,6 +1,6 @@
 const html = `<button type="button" class="js-open-modal" aria-expanded="true" aria-controls="dialog-content">Open a Dialog</button>
 
-<div id="dialog-content" class="dialog" aria-hidden="false" role="dialog" aria-labelledBy="h2" aria-modal="true">
+<div id="dialog-content" class="dialog" role="dialog" aria-labelledBy="h2" aria-modal="true">
```

Comment:
This aria-labeledBy points to the element instead of an ID, should it be labeled by the button id?

Reply:
Fixed that here: https://github.com/sparkbox/accessible-components/pull/134/commits/4405884bc8c2b6f034ae3674e36e822a8cbab797